### ISSUE TITLE
[typegen] Nullable params should be optional

### ISSUE THREAD

**Issue by qodunpob** [Quramy/ts-graphql-plugin#119]
Nullable GraphQL fields are also optional. I can make a method to update a certain entity in my API, and only pass some fields, instead of whole object if his params are not-required. But types generated by ts-graphql-plugin are required, even if they are nullable:

```typescript
export type SomeMutationVariables = {
    entityData: {
        id: number;
        param1: string | null;
        param2: number | null;
        param3: boolean | null;
    };
};
```

instead of

```typescript
export type SomeMutationVariables = {
    entityData: {
        id: number;
        param1?: string | null;
        param2?: number | null;
        param3?: boolean | null;
    };
};
```

**Comment by qodunpob** [comment]
May I get any feedback?

**Comment by Quramy** [comment]
@qodunpob Sorry for my late reply.

I fixed this and published as v2.0.2.

https://github.com/Quramy/ts-graphql-plugin/pull/182/files#diff-871d09f0641b968ebe1e1921f33875009e10651be09a4d178802df0d2ff2a635
